bower.json

```diff
@@ -29,7 +30,7 @@
     "type"
   ],
   "dependencies": {
-    "ramda": "^0.14.0"
+    "ramda": "0.1.x"
```

Comment:
You could commit a change on your local __robjens-1__ branch and push the branch to GitHub to update this pull request.